handlers.c

```diff
@@ -101,6 +101,9 @@ static FILE *get_pager(void)
     FILE *retfd = NULL;
     char *const emptyvec[1] = { NULL };
 
+    if (sm_globals.options.backend)
+        return stderr;
```

Comment:
`list` needs `stdout` here.

You might need a `backend_pager` arg to `get_pager`.